### ISSUE TITLE
Update Project and CI to Xcode 12 

### ISSUE THREAD

**Issue by LucianoPAlmeida** [SwifterSwift/SwifterSwift#939]
**Is your feature request related to a problem? Please describe.**
CI workflow still uses Xcode 11.7 https://github.com/SwifterSwift/SwifterSwift/blob/master/.github/workflows/CI.yml

**Describe the solution you'd like**
We should update both CI to use the latest available Xcode. 
Together all project things that needs updating also.

**Describe alternatives you've considered**

**Additional context**
None

- [x] I've read, understood, and done my best to follow the [Contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/tree/master/CONTRIBUTING.md) before opening this issue.

**Comment by jevonmao** [comment]
Should we use this [action](https://github.com/marketplace/actions/setup-xcode-version) to automatically use the newest version of Xcode available?

**Comment by LucianoPAlmeida** [comment]
Hi @jevonmao =] 
I think we could use the action if it makes the setup easier (which given how simple is our current workflow would not be required). Also IMO is not a problem to use a fixed version of Xcode, automatically updates may cause builds to fail when the Xcode upgrades and block current work, until a fix which can take sometime depending on size and availability of people to work on it. 
In short, we could use it but maybe just define a fixed version

**Comment by jevonmao** [comment]
100% I agree with you. Using an external action would make it more complicated since right now we can specify an Xcode version with 1 line of code. 
Can I fork this repo, make the change to update Xcode version, and open a PR?

**Comment by LucianoPAlmeida** [comment]
> Can I fork this repo, make the change to update Xcode version, and open a PR?

Sure! Feel free to tackle it :)

**Comment by jevonmao** [comment]
@LucianoPAlmeida I created a new branch. But the CI only runs on master branch, is there anyway I can test run the CI?

**Comment by LucianoPAlmeida** [comment]
When you open the PR CI will run on your branch :)
